scheduler/config-k8s.edn

```diff
@@ -108,16 +108,30 @@
  :rate-limit {:expire-minutes 1200 ; Expire unused rate limit entries after 20 hours.
               ; Keep these job-launch and job-submission values as they are for integration tests. Making them smaller can cause
               ; spurious failures, and making them larger will cause test_rate_limit_launching_jobs to skip itself.
-              :global-job-launch {:bucket-size 10000
-                                  :enforce? true
-                                  :tokens-replenished-per-minute 5000}
               :job-launch {:bucket-size 10000
                            :enforce? true
                            :tokens-replenished-per-minute 5000}
               :job-submission {:bucket-size 1000
                                :enforce? true
                                :tokens-replenished-per-minute 600}
               :user-limit-per-m 1000000}
+ :compute-cluster-launch-rate-limit {:expire-minutes 1200
+                                     :enforce? true
+                                     :matches
+                                     [{:compute-cluster-regex "^gke-1$"
+                                       :tbf-config {
+                                                    :bucket-size 1000
+                                                    :tokens-replenished-per-minute 2
+                                                    }}
+                                      {:compute-cluster-regex "^gke-2$"
+                                       :tbf-config {
+                                                    :bucket-size 1000
+                                                    :tokens-replenished-per-minute 4
+                                                    }}
+                                      {:compute-cluster-regex "^.*$"
+                                       :tbf-config {
+                                                    :bucket-size 1000
+                                                    :tokens-replenished-per-minute .0001}}]}
```

Comment:
@scrosby It seems like this would fit well in the compute cluster config `template`. @nsinkov Thoughts?

Reply:
See discussion above.